tests/EntityFrameworkCore.Projections.Generator.Tests/ProjectionExpressionGeneratorTests.cs

```diff
@@ -21,16 +19,14 @@ public ProjectionExpressionGeneratorTests(ITestOutputHelper testOutputHelper)
         }
 
         [Fact]
-        public void EmtpyCode_Noop()
+        public Task EmptyCode_Noop()
         {
             var compilation = CreateCompilation(@"
 class C { }
 ");
 
             var result = RunGenerator(compilation);
-
-            Assert.Empty(result.Diagnostics);
```

Comment:
```
Assert.Empty(result.Diagnostics);   
Assert.Empty(result.GeneratedTrees);
```

are not required since these will be included in snapshot

Reply:
Awesome, thanks!